src/transformers/models/clip/configuration_clip.py

```diff
@@ -106,10 +106,16 @@ def __init__(
         initializer_range=0.02,
         initializer_factor=1.0,
         pad_token_id=1,
```

Comment:
When I use the tokenizer from the `openai/clip` repository, the padding is done with `0`. But in HF CLIP tokenizer, it is `eos_token_id`, and the config here has default value `1`.

Although the padding is not used for pooling in clip text model, and should not affect the text-image similarity loss computation, I am afraid to change it here.